cardscan-base/src/main/java/com/getbouncer/cardscan/base/ScanActivityImpl.java

```diff
@@ -24,6 +24,7 @@
     public static final String SCAN_CARD_TEXT = "scanCardText";
     public static final String POSITION_CARD_TEXT = "positionCardText";
     public static final String API_KEY = "apiKey";
+    public static final String ENTER_CARD_MANUALLY_BUTTON = "enterCardManuallyButton";
```

Comment:
nit: rename this to `SHOW_ENTER_CARD_MANUALLY_BUTTON` so it's clear what this is doing.